src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -169,19 +169,22 @@ components:
       properties:
         deliveryClass:
           $ref: '#/components/schemas/DeliveryClass'
-        deliveryMethod:
-          $ref: '#/components/schemas/DeliveryMethod'
+        addressType:
```

Comment:
Minor comment here as well, @NathanRussellValtech . 
It was easier for me to understand what the `addressType` here refers to when I saw the ref pointing to `DeliveryAddressType`. I think it will be easier to understand if we have this field named as `deliveryAddressType` as well.

Reply:
Done 👍